graph_def_editor/graph.py

```diff
@@ -642,6 +310,22 @@ def to_graph_def(self, add_shapes = True):
     ret.versions.CopyFrom(self._passthrough_versions)
     for op in self.nodes:
       op.to_node_def(ret.node.add(), add_shapes)
+
+    # Pass through library without modifications for now.
```

Comment:
Comment appears to be outdated; code below modifies the function library.

Reply:
updated